### ISSUE TITLE
enums brackets not enforced with unexpected side effects

### ISSUE THREAD

**Issue by amammes** [raml-org/raml-js-parser-2#341]
```
types:
  myType:
    enum: value1, value2

/:
  post:
    body:
      type: myType
      example: value1
```

The parser says that the example has error "value should be one of:value1, value2". 
Clearly the enum definition is missing brackets. Why doesn't the parser complain about the missing brackets?
In order to satisfy the parser I have to define example: "value1, value2" (which is obviously not what I want), but imho this should not be the expected behaviour if the enum has a comma separated list of values even if the brackets are missing.

Should the parser believe that "value1, value2" is a single value instead of complaining about the brackets?

I am using api-workbench 0.8.33 installed via the atom packages menu and  "raml-1-parser": "0.2.26"

**Comment by KonstantinSviridov** [comment]
The issue is fixed in the `0.2.27` parser. It will be fixed in the next api-workbench release.

![enum-error](https://cloud.githubusercontent.com/assets/6206034/16661909/9fac2aa6-448e-11e6-8658-26e7641efff9.png)

**Comment by amammes** [comment]
Thanks for the info. I really don't see what value the brackets are adding to the enum syntax. We could easily just say enum: value1, value2. Everyone knows it's a list of items.
